.npmignore

```diff
@@ -25,7 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
-src
```

Comment:
Why was this removed? I do want `src/` to be ignored and `lib/` to be packaged upon npm publish.
See https://github.com/Glavin001/graphql-sequelize-crud/blob/master/package.json#L5